gsa/src/gmp/models/target.js

```diff
@@ -67,6 +67,8 @@ class Target extends Model {
       );
     }
 
+    ret.allow_simultaneous_ips = parseYesNo(element.allow_simultaneous_ips);
```

Comment:
Could we use camelCase for the variable at all places? I know it isn't consistent atm and will create some extra effort in the commands methods but only because we never got the time to refactor all models and dialogs to use camelCase instead of underscore.
```suggestion
    ret.allowSimultaneousIPs = parseYesNo(element.allow_simultaneous_ips);
```